altair/vegalite/v5/schema/channels.py

```diff
@@ -13,7 +13,7 @@
 
 from typing import TYPE_CHECKING, Any, Literal, Sequence, overload
 
-import pandas as pd
+from narwhals.dependencies import is_pandas_dataframe
```

Comment:
```suggestion
from narwhals.dependencies import is_pandas_dataframe as _is_pandas_dataframe
```
Avoiding adding to `__all__` (2/2)